neurots/generate/grower.py

```diff
@@ -298,58 +298,20 @@ def _convert_orientation2points(self, orientation, n_trees, distr, params):
         else:
             raise ValueError("Input orientation format is not correct!")
 
-        pts = self.soma_grower.add_points_from_orientations(orientations)
-        return pts
-
-    def _grow_trunks(self):
-        """Grow the trunks.
-
-        Generates the initial points of each tree, which depend on the selectedS
-        tree types and the soma surface. All the trees start growing from the surface
-        of the soma. The outgrowth direction is either specified in the input parameters,
-        as ``parameters['type']['orientation']`` or it is randomly chosen according to the
-        biological distribution of trunks on the soma surface if ``orientation`` is ``None``.
-        """
-        tree_types = self.input_parameters["grow_types"]
-
-        legacy_mode = not isinstance(self.input_parameters[tree_types[0]]["orientation"], dict)
-
-        if not legacy_mode:
-
-            trunk_orientations_manager = self._trunk_orientations_class(
-                soma=self.soma_grower.soma,
-                parameters=self.input_parameters,
-                distributions=self.input_distributions,
-                context=self.context,
-                rng=self._rng,
-            )
-
-        for type_of_tree in tree_types:
+        return self.soma_grower.add_points_from_orientations(orientations)
 
+    def _simple_grow_trunks(self):
+        """Simple _grow_trunks function."""
+        for type_of_tree in self.input_parameters["grow_types"]:
             params = self.input_parameters[type_of_tree]
             distr = self.input_distributions[type_of_tree]
 
-            if legacy_mode:
```

Comment:
@eleftherioszisis to check if it's ok to remove legacy

Reply:
it's not, many test still rely on this legacy  mode, so I reverted